lib/spack/docs/environments.rst

```diff
@@ -971,6 +971,28 @@ For example, the following environment has three root packages:
 This allows for a much-needed reduction in redundancy between packages
 and constraints.
 
+-----------------
+Modifying Environment Variables
+-----------------
+
+Spack Environments can modify environment variables when activated.  The environment can be
+configured to set, unset, prepend, or append using ``env_vars`` option in the ``spack.yaml``
+file:
```

Comment:
Please add that it can also be set with an `env_vars.yaml` file. It is a proper config scope so it can exist outside the `spack.yaml`

Reply:
My statement is actually wrong.  I'll put up a PR to fix it in a minute.